### PR TITLE
Replace Pull data source oddrns with synthetical

### PR DESCRIPTION
**Pull request by DementevNikita**

### CODE REVIEW

File: src/main/java/com/provectus/oddplatform/puller/task/PullerTask.java [hunk 1]
```diff
@@ -13,14 +17,31 @@
     private final AdapterRestService adapterRestService;
     private final PlatformRestService platformRestService;
 
+    private final Generator generator = new Generator();
```
Comment (DementevNikita) [inline src/main/java/com/provectus/oddplatform/puller/task/PullerTask.java]: Can this work concurrently?

File: src/main/java/com/provectus/oddplatform/puller/task/PullerTask.java [hunk 2]
```diff
@@ -13,14 +17,31 @@
...
+            .getEntities(dataSourceDto.getEndpoint())
+            .flatMap(this::substituteOddrn)
             .flatMap(platformRestService::ingest)
             .doOnError(t -> log.error(t.getMessage()))
-            .subscribe(r -> log.info("Puller task has been completed. Data Source: {}", dataSourceDto.getOddrn()));
+            .subscribe(r -> log.info("Puller task has been completed. Data Source: {}", dataSourceDto.getId()));
+    }
+
+    private Mono<DataEntityList> substituteOddrn(final DataEntityList dataEntityList) {
+        final ODDPlatformDataSourcePath oddrnPath = ODDPlatformDataSourcePath.builder()
```
Comment (DementevNikita) [inline src/main/java/com/provectus/oddplatform/puller/task/PullerTask.java]: Each time OddrnPath is created. Move out of the run stacktrace or cache

Comment (damirabdul) [reply src/main/java/com/provectus/oddplatform/puller/task/PullerTask.java]: may be create it in constructor? It depends only on final field

Comment (DementevNikita) [reply src/main/java/com/provectus/oddplatform/puller/task/PullerTask.java]: Moved it out of the PullerTask scope

**Review by damirabdul — APPROVED** [review]

**Pull request merged**